compose/aarch64.yml

```diff
@@ -0,0 +1,11 @@
+# Overrides for AArch64 processors, until we get official images:
+
+services:
+  cardano-db-sync:
+    image: ghcr.io/input-output-hk/ogmios-tracker/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-sancho-4-0-0}
+  cardano-node:
+    image: ghcr.io/input-output-hk/ogmios-tracker/cardano-node:${CARDANO_NODE_VERSION:-8.8.0-pre}
+  ogmios:
+    image: ghcr.io/input-output-hk/ogmios-tracker/ogmios:v${OGMIOS_VERSION:-6.1.0}
```

Comment:
We have to remember to update these versions here, if @mirceahasegan's PRs get merged first (#1181, #1184)